yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/MultiServerConnectionFactory.java

```diff
@@ -64,25 +69,34 @@ public enum ConnectionSelectionPolicy { DEFAULT, ALTERNATE }
     private final List<YamlRunner.YamlConnectionFactory> alternateFactories;
 
     public MultiServerConnectionFactory(@Nonnull final YamlRunner.YamlConnectionFactory defaultFactory,
-                                        @Nullable final List<YamlRunner.YamlConnectionFactory> alternateFactories) {
+                                        @Nonnull final List<YamlRunner.YamlConnectionFactory> alternateFactories) {
```

Comment:
making alternateFactories Nullable would probably mean doing the same at line 48

Reply:
Line 48 is in the middle of a comment, so I don't think that's what you meant?
Did you mean line 68?

Reply:
Yes. line numbers changed with the repo move